pages/api/v1/bounty/curator/getCuratorBio.ts

```
@@ -0,0 +1,73 @@
+import apiErrorWithStatusCode from '~src/util/apiErrorWithStatusCode';
+
+interface Args {
+	network: string;
+	userId: number;
+}
+
+export async function getCuratorBio({ network, userId }: Args): Promise<IApiResponse<{ curatorBio: string }>> {
+	try {
+		if (!network || !isValidNetwork(network)) throw apiErrorWithStatusCode(messages.INVALID_NETWORK, 400);
+
+		if (isNaN(userId)) throw apiErrorWithStatusCode(messages.INVALID_PARAMS, 400);
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)